docs/ManualDeploymentGuideScalarDLonAWS.md

```diff
@@ -0,0 +1,233 @@
+# Deploy Scalar DL on AWS
+
+Scalar DL is a database-agnostic distributed ledger middleware containerized with Docker.
+It can be deployed on various platforms and is recommended to be deployed on managed services for production to achieve high availability and scalability, and maintainability.
+This guide shows you how to manually deploy Scalar DL on a managed database service and a managed Kubernetes service in Amazon Web Services (AWS) as a starting point for deploying Scalar DL for production.
+
+## What we create
+
+![image](images/network_diagram_eks.png)
+
+In this guide, we will create the following components.
+
+* A VPC with NAT gateway
+* An EKS cluster with two Kubernetes node groups
+* A managed database service
+    * DynamoDB
+* A Bastion instance with a public IP
+* Amazon CloudWatch
+
+## Step 1. Configure your network
+
+Configure a secure network with your organizational or application standards. Scalar DL handles highly sensitive data of your application, so you should create a highly secure network for production.
+This section shows how to configure a secure network for Scalar DL deployments.
+
+### Requirements
+ 
+* You must create VPC with NAT gateways on private networks. NAT gateway is necessary to enable internet access for Kubernetes node group subnets.
+* You must create at least 2 subnets for the EKS cluster in different availability zones. This is mandatory to create an EKS cluster.
+* You must create subnets with the prefix at least `/24` for the Kubernetes cluster to work without issues even after scaling.
+
+### Recommendations
+
+* You should create private subnets for the Kubernetes cluster for production.
+* You should create a bastion server to manage the Kubernetes cluster.
+* You should create 3 subnets in 3 availability zones for the Kubernetes cluster for higher availability.
+* You should create public subnets for the Kubernetes cluster if you want to place envoy LoadBalancer in the public subnet for testing purposes.
+
+### Steps
+
+* Create an Amazon VPC on the basis of [AWS official guide](https://docs.aws.amazon.com/batch/latest/userguide/create-public-private-vpc.html) with the above requirements and recommendations.
```

Comment:
This URL is for a batch service, so it looks a bit sensitive.

Reply:
Thank you. Changed URL in a30f2e4